### DISCUSSION TITLE
Testing out question / answer enabled categories

### DISCUSSION THREAD

**Discussion by w00fz** [getgrav/grav#3100] [Help] [answered]
This is a question / answer test discussion. Being under the **Help** and **Development** categories, It enables people to ask questions, suggest answers, and vote on the best suggested answer, unlike **Open ended discussion` formats such as  #3097.

So I figured we would give this a go!

**Comment by paulhibbitts** [comment, marked as answer]
Awesome, here is one possible answer: Yes 🙂

**Reply by w00fz** [reply]
You win 🙂

**Reply by w00fz** [reply]
Actually really nice
